docs/book/src/migration/v2vsv3.md

```diff
@@ -41,7 +41,7 @@ Projects scaffolded with Kubebuilder v3 will use the `go.kubebuilder.io/v3` plug
   * A new option to create the projects using ComponentConfig is introduced. For more info see its [enhancement proposal][enhancement proposal] and the [Component config tutorial][component-config-tutorial]
   * Manager manifests now use `SecurityContext` to address security concerns. More info: [#1637][issue-1637] 
 - Misc:
-  * Support for [controller-tools][controller-tools] `v0.4.1` (for `go/v2` it is `v0.3.0` and previously it was `v0.2.5`) 
+  * Support for [controller-tools][controller-tools] `v0.5.0` (for `go/v2` it is `v0.3.0` and previously it was `v0.2.5`) 
```

Comment:
is this version bump going to be back ported ? 
If not, can we directly bump controller-tolls to 0.6.0 and controller-runtime to 0.9.0. That would bring in k8s 1.21 support for projects scaffolded with v3 plugin.

Reply:
I can definitely add bumps - controller-tools v0.6.0 had yet to be published at the time of PR. Is the preference for including it in this PR or should I start a separate one to include those bumps?

Reply:
Two PRs is fine with me.

Reply:
I'll create a second PR today at some point today to cover this (assuming following caveat is resolved). I did an initial pass updating controller-tools to v0.6.0 and controller-runtime to v0.9.0. The controller-runtime update seemed to cause some issues, but I haven't had a chance to investigate yet.

Reply:
#2208 will handle this.

Reply:
@estroz sg!